New dbt cloud IA (#2932)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-sso-azure-active-directory.md

4. Configure the **Redirect URI**. The table below shows the appropriate
   Redirect URI values for single-tenant and multi-tenant deployments. For most

>    enterprise use-cases, you will want to use the single-tenant Redirect URI.

Author: ```suggestion
   enterprise use-cases, you will want to use the single-tenant Redirect URI. Replace `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan.
```

| Application Type | Redirect URI |
| ----- | ----- |

> | Single-Tenant _(recommended)_ | `https://cloud.getdbt.com/complete/azure_single_tenant` |

Author: ```suggestion
| Single-Tenant _(recommended)_ | `https://YOUR_ACCESS_URL/complete/azure_single_tenant` |
```

> | Multi-Tenant | `https://cloud.getdbt.com/complete/azure_multi_tenant` |

Author: ```suggestion
| Multi-Tenant | `https://YOUR_ACCESS_URL/complete/azure_multi_tenant` |
```

> *Note:* If your dbt account instance is a VPC deployment or is based [outside the US](/docs/cloud/about-cloud/regions-ip-addresses), your login URL will use the domain supplied to you by your dbt Labs account team, instead of the domain `cloud.getdbt.com`.

Author: ```suggestion
```

### Supplying credentials

> 24. Go to [Settings](https://cloud.getdbt.com/next/settings/profile). On the left side, select **Single Sign On** under **Account Settings**.

Author: ```suggestion
24. Click the gear icon at the top right and select **Profile settings**. To the left, select **Single Sign On** under **Account Settings**.
```

| **Client&nbsp;Secret** | Paste the **Client Secret** (remember to use the Secret Value instead of the Secret ID) recorded in the steps above |
| **Tenant&nbsp;ID** | Paste the **Directory (tenant ID)** recorded in the steps above |
| **Domain** | Enter the domain name for your Azure directory (eg. `fishtownanalytics.com`). Only users with accounts in this directory with this primary domain will be able to log into the dbt Cloud application. Optionally, you may specify a CSV of domains which are _all_ authorized to access your dbt Cloud account (eg. `fishtownanalytics.com, fishtowndata.com`) Ensure that the domain(s) match the values configured on user accounts in Azure |

> | **Slug** | Enter your desired login slug. Users will be able to log into dbt Cloud by navigating to `https://cloud.getdbt.com/enterprise-login/<login-slug>`. Login slugs must be unique across all dbt Cloud accounts, so pick a slug that uniquely identifies your company. |

Author: ```suggestion
| **Slug** | Enter your desired login slug. Users will be able to log into dbt Cloud by navigating to `https://YOUR_ACCESS_URL/enterprise-login/LOGIN_SLUG`, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/deploy/regions-ip-addresses) for your region and plan. Login slugs must be unique across all dbt Cloud accounts, so pick a slug that uniquely identifies your company. |
```

    here, you can navigate to the login URL generated for your account's _slug_ to
    test logging in with Azure AD.

:::success Logging in

> *Note:* If your dbt account instance is a VPC deployment or is [based outside the US](/docs/cloud/about-cloud/regions-ip-addresses), your login URL will use the domain supplied to you by your dbt Labs account team, instead of the domain `cloud.getdbt.com`.

Author: ```suggestion
```

    here, you can navigate to the login URL generated for your account's _slug_ to
    test logging in with Azure AD.

> :::success Logging in

Author: ```suggestion
<Snippet src="login_url_note" />
```
